packages/flutter/lib/src/cupertino/app.dart

```diff
@@ -520,6 +520,7 @@ class _CupertinoAppState extends State<CupertinoApp> {
         CupertinoIcons.search,
         size: 28.0,
         color: CupertinoColors.white,
+        semanticLabel: 'Enter select widget mode',
```

Comment:
We might want to have this string in one place rather than duplicate it, but I'm not sure where it should go?

Reply:
Please follow the 'Adding a new string to localizations" instructions [here](https://github.com/flutter/flutter/tree/master/packages/flutter_localizations)


This `cupertino.app.dart` class imports Cupertino's `localizations.dart`, which contains `CupertinoLocalizations`. 

https://github.com/flutter/flutter/blob/05914e7738be72669ef452badc00e98e43b6103c/packages/flutter/lib/src/cupertino/app.dart#L11

Reply:
I recommend filing a Separate PR for adding a new string in https://github.com/flutter/flutter/tree/master/packages/flutter_localizations.

Once`flutter_localizations` PR is merged, you can update this PR to get the string.

Reply:
@TahaTesser I don't believe that's necessary - Fluter DevTools (which this is effectively part of) is only available in English, and all strings are hardcoded.

Reply:
That's a good point. I didn't realize it's just the inspector button -> `_inspectorSelectButtonBuilder`.

Reply:
Although this button is part of the `CupertinoApp`, I would expect all elements to support translated semantic strings.
If you're testing the app in a non-English locale, the inspector button would be out of place but I understand it might not be necessary for debugging.

Reply:
> We might want to have this string in one place rather than duplicate it, but I'm not sure where it should go?

In that case, I think it's fine to have this string here.

Reply:
Can this be in WidgetInspector widget instead? so that you don't need to duplicate the string